clients/src/test/java/org/apache/kafka/test/TestSslUtils.java

```diff
@@ -89,7 +89,7 @@ public static X509Certificate generateCertificate(String dn, KeyPair pair,
 
     public static KeyPair generateKeyPair(String algorithm) throws NoSuchAlgorithmException {
         KeyPairGenerator keyGen = KeyPairGenerator.getInstance(algorithm);
-        keyGen.initialize(1024);
+        keyGen.initialize(2048);
```

Comment:
Sorry this was committed by mistake. It's [KAFKA-7789](https://issues.apache.org/jira/browse/KAFKA-7789), which I fixed in #6096, but which no one seems to have reviewed yet. It would be great if this could be reviewed because I can't run the tests without this.

Reply:
Ah. I merged the PR in KAFKA-7789.